README.md

```diff
@@ -21,6 +20,16 @@ You can peruse the API reference documentation as:
 * [Swagger UI](https://vippsas.github.io/vipps-qr-api/)
 * [ReDoc](https://vippsas.github.io/vipps-qr-api/redoc.html)
 
+💥 DRAFT: This is unfinished work and subject to change. 💥
+## Merchant Redirect QR
+Merchant Redirect QRs will be a product where merchants can make QRs that simply point to their own websites.
+This solution is meant for more permanent usecases such as stickers at the counter, billboards, TV-commercials etc. where you want the customer to be able to be redirected at a product page directly from scanning the QR.
+
+![uml diagram](images/uml-of-merchant-flow.png)
+
+This endpoint will make it possible to generate a vipps-branded QR with the ability to change the url it points to. That is possible because the QR points to a url on the format `qr.vipps.no/r/abc123` and what that url redirects to simply has to be updated in Vipps´ database. These QRs will be scannable from both vipps QR scanner and native camera scanner, and will always redirect the user to the targetUrl that you decided.
+
+In the future, this solution will be compatible with express checkout, or "vipps hurtigkasse". Such that the user will be able to buy the product directly from scanning the QR. But as of now, this solution will only redirect to the product page that you choose.
```

Comment:
Maybe remove this at it might confuse merchants. The one time payment QRs already works with express checkout, the Merchant Redirect QRs wont do anything else than a redirect back to merchants

Reply:
@kaffikaffi

Reply:
Sounds good!